Makefile

```diff
@@ -32,15 +29,15 @@ inplace:
 	$(PYTHON) setup.py build_ext -i
 
 test-code:
-	$(NOSETESTS) -s nilearn $(NOSETESTS_OPTIONS)
+	python -m pytest --pyargs nilearn --cov=nilearn
```

Comment:
I think that's fine, because they can run pytest directly for that. & Best practices dictate to install Nilearn using pip in editable mode anyway for the paths to work properly during development.

Personally I am not a big fan of so many nested files & scripts (make file, test_script) but that is a a discussion for another PR.

Reply:
> Personally I am not a big fan of so many nested files & scripts (make file, test_script) but that is a a discussion for another PR.

:+1: